cloudinary-core/src/test/java/com/cloudinary/test/CloudinaryTest.java

```diff
@@ -1468,6 +1469,8 @@ private void setRandomValue(Random rand, Field field, Object instance) throws Il
             field.set(instance, rand.nextInt());
         } else if (fieldType.equals(long.class) || fieldType.equals(Long.class)) {
             field.set(instance, rand.nextLong());
+        } else if (field.get(instance) instanceof List) {
+            field.set(instance, Collections.singletonList(cloudinary.randomPublicId()));
```

Comment:
@YomesInc can you please explain this change, I was not sure what it's related to